pype/plugins/nuke/publish/collect_review.py

```diff
@@ -23,6 +26,21 @@ def process(self, instance):
         if not node["review"].value():
             return
 
+        # Add audio to instance if it exists.
+        try:
+            version = pype.api.get_latest_version(
+                instance.context.data["assetEntity"]["name"], "audioMain"
```

Comment:
asset name should be a class attribute to allow overriding it in the presets. Different studios might have different convention on naming here.

Reply:
You mean `audioMain`?

Reply:
yes. `audioMain`. it's entirely possible that a studio would want to bring in `audioVoices` or another subset.